src/function.rs

```diff
@@ -1,9 +1,12 @@
+use semver::VersionReq;
+
 use {
   super::*,
   heck::{
     ToKebabCase, ToLowerCamelCase, ToShoutyKebabCase, ToShoutySnakeCase, ToSnakeCase, ToTitleCase,
     ToUpperCamelCase,
   },
+  semver::Version,
```

Comment:
You can put everything into the same use block:

```suggestion
use {
  super::*,
  heck::{
    ToKebabCase, ToLowerCamelCase, ToShoutyKebabCase, ToShoutySnakeCase, ToSnakeCase, ToTitleCase,
    ToUpperCamelCase,
  },
  semver::{Version, VersionReq},
```

Reply:
thanks for this suggestion -- combined all the imports!